components/heroes/ExclusiveEquipmentSection.tsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import { ExclusiveEquipment } from "../../types/hero";
+
+export default function ExclusiveEquipmentSection({
+  equipment,
+}: {
+  equipment: ExclusiveEquipment;
+}) {
+  return (
+    <>
+      <div className="grid grid-cols-12 items-center mt-2 mb-2">
+        <div className="col-span-12 text-center sm:col-span-1">
+          <div className="mb-3">
+            <img
+              src={"/equipment/" + equipment.name + ".png"}
+              className="inline"
+              width={70}
+              height={70}
+            ></img>
+          </div>
+        </div>
+        <div className="col-span-12 text-center sm:col-span-11 sm:text-left">
+          <p className="text-2xl">
+            {`Exclusive Equipment:  ${equipment.name}`}
+          </p>
+          <p className="whitespace-pre-line">{equipment.effect}</p>
+        </div>
+      </div>
+    </>
+  );
+}
```

Comment:
I see why you did this, but in this case I would think the existing `isExclusive` would have been sufficient to switch between the two different implementations. Is there something im missing?

Reply:
Equipment become so much different than Exclusive, so i decided to make two simpler components. They render in different places, render different props, so i decided to split it.